index.js

```diff
@@ -54,6 +47,18 @@ function parse(str) {
     config.ssl = true;
   }
 
+  ['db', 'database', 'encoding', 'client_encoding', 'host', 'port', 'user', 'password', 'ssl']
+  .forEach(function(key) {
+    delete result.query[key];
+  });
+
+  Object.getOwnPropertyNames(result.query).forEach(function(key) {
```

Comment:
`Object.getOwnPropertyNames` is [not available](http://node.green/#Object-static-methods-accept-primitives) in 0.10, so probably change it to `Object.keys`?